lips/lip-1.mdx

```diff
@@ -232,16 +283,17 @@ For a travel rule data exchange, the [command_type](basic_building_blocks.md#com
 |-------	    |------	|-----------	|-------------	|
 | _ObjectType   | str  | Y             | The fixed string `PaymentCommand` |
 | payment| [`PaymentObject`](#paymentobject) | Y | contains a `PaymentObject` that either creates a new payment or updates an existing payment. Note that strict validity check apply when updating payments, that are listed in the section below describing these objects. An invalid update or initial payment object results in a command error
-| _creates_versions | list of str |  Y | Must be a list containing a single str representing the version of the new or updated `PaymentObject` resulting from the success of this payment command. A list with any other number of items results in a command error.  This string must be a unique random string between this pair of VASPs and is used to represent the version of the item created. These should be at least 16 bytes long and encoded to string in hexadecimal notation using characters in the range[A-Za-z0-9] |	
-| _dependencies | list of str | Y | Can be an empty list or a list containing a single previous version. If the list is empty this payment command defines a new payment. If the list contains one item, then this command updates the shared `PaymentObject` with the given version. It is an error to include more versions, and it results in a command error response.  The value in this field must match a version previously specified by the `_creates_versions` parameter on a prior command. |
+| _reads | JSON object map |  Y | Must be an object containing mappings of `reference_id` to latest version as represented by the `command_version` of the latest command which successfully mutated the object referenced by the `reference_id`. The value in this field must match a version previously specified by the `_writes` parameter on a prior command for this payment object.  For a payment command, only zero or one `_reads` values should be specified since payments are only dependent upon at most prior version of an object. A list with any other number of items results in a command error.  If the list is empty this payment command defines a new payment. If the list contains one item, then this command updates the shared `PaymentObject`. |	
```

Comment:
want to comment on `OffChainErrorObject` but it's doable.
question is how to distinguish "command_error" and "protocol_error" ?

Reply:
> want to comment on `OffChainErrorObject` but it's not doable.
> question is how to distinguish "command_error" and "protocol_error" ?

Reply:
George is going to unify our errors a bit more and then we can update this with all error codes.  We may end up removing one of the two types and merging them into a single erorr type